tools/mypy-onnx.py

```diff
@@ -1,25 +0,0 @@
-#!/usr/bin/env python
-
-# SPDX-License-Identifier: Apache-2.0
-
-import os
-import subprocess
-
-
-def main() -> None:
-    try:
-        root_folder = os.path.realpath(
-            os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
-        )
-        os.chdir(root_folder)
-        # Use --no-site-packages to prevent mypy catching other typecheck errors which are not related to ONNX itself
-        subprocess.check_call(["mypy", ".", "--no-site-packages"])
-
-        exit(0)
-    except subprocess.CalledProcessError:
-        # Catch this exception because we don't want it to output a backtrace that would clutter the mypy output
-        exit(1)
-
-
-if __name__ == "__main__":
-    main()
```

Comment:
Is this file no longer needed because of this PR https://github.com/onnx/onnx/pull/4654?

Reply:
I think so.